pkg/scheduler/capabilities/volumebinding/volume_binding.go

```diff
@@ -22,6 +22,7 @@ import (
 	"fmt"
 	"sync"
 	"time"
+	"volcano.sh/volcano/cmd/scheduler/app/options"
```

Comment:
go-lint format

Reply:
Hmm. Some CIs for this PR are faield. I am  try to fix the errors.